### ISSUE TITLE
Nova preview cache

### ISSUE THREAD

**Issue by war1an** [tommasongr/nova-vue#32]
This is not directly linked with the extension. But I'm having issue with the preview tab inside Nova. I open the tab at `localhost:8080` and at first I can see the page and everything works like a charm. But at some point.. the window starts showing old code or the changes don't take effect.

After searching on the web, I found that the issue is related to how Safari does its caching. If I open `localhost:8080` using Chrome everything looks like It should.

Have you encountered this issue or you are not using the Nova preview tab?

**Comment by tommasongr** [comment]
Hi there! I don't use Nova preview on a daily basis but I do use Safari as my main dev browser along with Firefox.

I can confirm you that in Safari there are some differences with how cache control behave. Therefore, for example, on every NuxtJS project I always add these lines to the build settings. I found out that it solves all the problems with cache. Maybe you can try to set this Headers too.

```js
    build: {
        // Fix infinite reloading loop in Safari
        devMiddleware: {
            headers: {
                'Cache-Control': 'no-store',
                Vary: '*',
            },
        },
    },
```

**Comment by command-tab** [comment]
FWIW, this Safari infinite reload issue was fixed in Nuxt 2.15.4:
https://github.com/nuxt/nuxt.js/issues/3828#issuecomment-805134947
https://github.com/nuxt/nuxt.js/pull/9088
